### ISSUE TITLE
QUploader component's background colour changes slightly when 'text-color' prop is set

### ISSUE THREAD

**Issue by vijay-singh-16** [quasarframework/quasar#12515]
### When 'text-color' prop is NOT set:

![no-text-color](https://user-images.githubusercontent.com/60729120/154657596-4f985598-d8d9-4496-a433-1713f43596db.jpg)

### When 'text-color' prop is set:

![grey-5-text-color](https://user-images.githubusercontent.com/60729120/154657609-55ee9e6e-5aa3-4753-930c-55967fa3035f.jpg)

This happens with any 'color' prop value.

### What did you expect to happen?

The background colour should not change when 'text-color' prop is set.

### Reproduction URL

https://codepen.io/vijay-singh-16/pen/MWOQegE?editors=101

### How to reproduce?

Refer reproduction URL.

### Flavour

Vue CLI Plugin (vue-cli-plugin-quasar)

### Areas

Components (quasar)

### Platforms/Browsers

Firefox, Chrome, Microsoft Edge

**Comment by Saeid-Za** [comment]
Hello There!
I've created a pull-request that should resolve this issue, until then, there is a workaround, put this chunk of css into your global style file.
```scss
  .q-uploader__header::before{
    background: unset !important;
  }
```

**Comment by rstoenescu** [comment]
Fix will be available in Quasar v2.7.5 and 1.19.4
